src/components/v2/V2Dashboard/index.tsx

```diff
@@ -44,16 +46,38 @@ export default function V2Dashboard() {
     projectId,
   })
 
+  const { data: queuedFundingCycle } = useProjectQueuedFundingCycle({
+    projectId,
+  })
+
   const { data: payoutSplits } = useProjectSplits({
     projectId,
     splitGroup: ETH_PAYOUT_SPLIT_GROUP,
     domain: fundingCycle?.configuration?.toString(),
   })
-
   const { data: terminals } = useProjectTerminals({
+    // useMemo here
     projectId,
   })
 
+  const { data: distributionLimit } = useProjectDistributionLimit({
+    projectId,
+    domain: fundingCycle?.configuration?.toString(),
+    terminal: terminals ? terminals[0] : undefined, //TODO: make primaryTerminalOf hook and use it
+  })
+
+  const { data: queuedDistributionLimit } = useProjectDistributionLimit({
+    projectId,
+    domain: queuedFundingCycle?.configuration?.toString(),
+    terminal: terminals ? terminals[0] : '',
```

Comment:
```suggestion
    terminal: terminals?.[0],
```

Reply:
Done

Reply:
Sweet @aeolianeth . This was causing some errors because when `fundAccessContraints` hadn't been loaded into the redux state. So I made it load this whole object, with the `distributionLimitCurrency` (for queued FC if it exists). ( see `V2ReconfigureModal/index:109` onwards

Reply:
all im sayin is `terminals ? terminals[0]` is equivalent to `terminals?.[0]`